ffsync/ordered_id.go

```diff
@@ -83,3 +89,76 @@ func (m *memoryIdGenerator) NextId(namespace string) (OrderedId, error) {
 	nextId := atomic.AddUint64((*uint64)(m.counters[namespace]), 1)
 	return (*Uint64OrderedId)(&nextId), nil
 }
+
+func NewETCDOrderedIdGenerator() (OrderedIdGenerator, error) {
+	etcdHost := helpers.GetEnv("ETCD_HOST", "localhost")
+	etcdPort := helpers.GetEnv("ETCD_PORT", "2379")
```

Comment:
Feels weird to use env variables like this here. Also feels like this stuff should be params

Reply:
Does it make sense to have `NewETCDTaskMetadataManager` in scheduling.taskmetadata.go pull the variables from the environment then pass them as parameters?